spec/features/users_can_create_involuntary_conversion_projects_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "rails_helper"
+
+RSpec.feature "Users can create new involuntary conversion projects" do
+  let(:regional_delivery_officer) { create(:user, :regional_delivery_officer) }
+
+  before do
+    sign_in_with_user(regional_delivery_officer)
+    visit conversion_involuntary_new_path
+  end
+
+  context "when the URN and UKPRN are valid" do
+    let(:urn) { 123456 }
+    let(:ukprn) { 10061021 }
+    let(:two_weeks_ago) { Date.today - 2.weeks }
+
+    before { mock_successful_api_responses(urn: urn, ukprn: ukprn) }
+
+    scenario "a new project is created" do
+      expect(page).to have_content(I18n.t("conversion_project.involuntary.new.title"))
+      fill_in "School URN", with: urn
+      fill_in "Incoming trust UK Provider Reference Number (UKPRN)", with: ukprn
+
+      within("#provisional-conversion-date") do
+        completion_date = Date.today + 1.year
+        fill_in "Month", with: completion_date.month
+        fill_in "Year", with: completion_date.year
+      end
+
+      fill_in "School SharePoint link", with: "https://educationgovuk-my.sharepoint.com/school-folder"
+      fill_in "Trust SharePoint link", with: "https://educationgovuk-my.sharepoint.com/trust-folder"
+
+      within("#advisory-board-date") do
+        fill_in "Day", with: two_weeks_ago.day
+        fill_in "Month", with: two_weeks_ago.month
+        fill_in "Year", with: two_weeks_ago.year
+      end
+
+      fill_in "Advisory board conditions", with: "This school must:\n1. Do this\n2. And that"
+
+      fill_in "Handover comments", with: "A new handover comment"
+
+      click_button("Continue")
+
+      expect(page).to have_content(I18n.t("project.show.title"))
+      expect(page).to have_content("Involuntary task")
+
+      click_on("Project information")
+
+      expect(page).to have_content(two_weeks_ago.to_formatted_s(:govuk))
+    end
+  end
+
+  context "when some required project information is not valid" do
```

Comment:
We have been putting the `invalid` tests in the request specs so we can keep the test suite as fast as possible. I think we are trying to reserve feature specs for the happy path. 

https://github.com/DFE-Digital/dfe-complete-conversions-transfers-and-changes/blob/5d1f9c4552a94a8d8341b8eaa8d63dbd301c7fc4/spec/requests/notes_controller_spec.rb#L74

Reply:
I added this test because without it, coverage was less than 100%. I've moved a similar test into requests and the coverage is still 100% but it feels a bit like a useless test. Not sure how to keep coverage happy without this slightly pointless test.